cmd/ctr/app/main.go

```diff
@@ -95,7 +95,7 @@ containerd CLI
 		cli.StringFlag{
 			Name:   "namespace, n",
 			Usage:  "namespace to use with commands",
-			Value:  namespaces.Default,
+			Value:  "k8s.io",
```

Comment:
Please don’t change the default. Ctr isn’t just made for k8s.

Reply:
Ok, we will remove this change and keep the others